jujugui/static/gui/src/app/components/status/test-status.js

```diff
@@ -0,0 +1,33 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+
+'use strict';
+
+var juju = {components: {}}; // eslint-disable-line no-unused-vars
+
+describe('Status', function() {
+  beforeAll(function(done) {
+    // By loading this file it adds the component to the juju components.
+    YUI().use('status', function() {
+      done();
+    });
+  });
+
+  it('can render', () => {
+    const renderer = jsTestUtils.shallowRender(
```

Comment:
I'd abstract out a render(props) method that can be reused by all tests that will be implemented here.